docarray/document/mixins/_property.py

```diff
@@ -110,23 +110,14 @@ def uri(self, value: str):
         self._data.uri = value
 
     @property
-    def tags(self) -> Optional[Dict[str, 'StructValueType']]:
+    def tags(self) -> Optional[Dict[str, Any]]:
         self._data._set_default_value_if_none('tags')
         return self._data.tags
 
     @tags.setter
-    def tags(self, value: Dict[str, 'StructValueType']):
+    def tags(self, value: Dict[str, Any]):
         self._data.tags = value
 
-    @property
-    def _metadata(self) -> Optional[Dict[str, 'StructValueType']]:
```

Comment:
why are you changing this here?

Reply:
`_property.py` is auto-generated using `scripts/gen_doc_property_mixin.py`. This line is the same as the code in the main branch https://github.com/docarray/docarray/blob/main/docarray/document/mixins/_property.py.

Reply:
okey,

Reply:
@JoanFM sorry, my mistake. I double-checked, and getter and setter for `_metadata` are missing in the version of `_property.py` in my branch because the script `cripts/gen_doc_property_mixin.py` has the following code

```python
    for f in fields(DocumentData):
        if f.name.startswith('_'):
            continue
   ....
```

and therefore it skips all fields starting with `_`. I do not know why you have those getter and setter in the `main` branch. I can remove the `if` in the script, but in that case the script would generate also getter and setter for other fields of `Document` starting with `_`, such as `_reference_doc`. Alternatively, I may had the getter and setter for `_metadata` manually.

What do you prefer? Any other idea?

Reply:
Let's add them manually